### PR TITLE
CORE-8430 Restore App Stats table sorting.

### PR DESCRIPTION
**Pull request by psarando**
This PR will restore App Stats table sorting for the new React/material-ui components.

These changes are largely based on https://material-ui.com/demos/tables/#sorting-amp-selecting

### CODE REVIEW

File: react-components/src/apps/admin/AppStats.js [hunk 1]
```diff
@@ -211,22 +294,28 @@ class AppStats extends Component {
...
-                                           numeric>{getMessage("lastCompleted")}</TableCell>
-                                <TableCell className={classes.statTableHead}
-                                           numeric>{getMessage("lastUsed")}</TableCell>
+                                {columnData.map(column => (
+                                    <TableCell key={column.name}
+                                               numeric={column.numeric}
+                                               sortDirection={orderBy === column.name ? order : false}
+                                    >
+                                        <Tooltip
+                                            title="Sort"
```
Comment (ash3rz) [inline react-components/src/apps/admin/AppStats.js]: Should this string be in a messages file for i18n?

Comment (psarando) [reply react-components/src/apps/admin/AppStats.js]: Yes, good catch 👍

**Comment by sri2k1us** [comment]
LGTM 🚢

**Comment by psarando** [comment]
Thanks for the reviews! 👍

**Pull request merged**